README.md

```diff
@@ -1,3 +1,4 @@
 # Giraffe
 about Giraffes!
 Here is a sample line
+It feels good!
```

Comment:
It's okay,  but if You Capitalize "f" in "feels" it would be great!

Reply:
okay, done!